modules/claims_api/lib/bgs_service/person_web_service.rb

```diff
@@ -7,15 +7,33 @@ def bean_name
     end
 
     def find_dependents_by_ptcpnt_id(id)
-      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
-        <ptcpntId />
-      EOXML
-
-      { ptcpntId: id }.each do |k, v|
-        body.xpath("./*[local-name()='#{k}']").first.content = v
+      builder = Nokogiri::XML::Builder.new do
+        ptcpntId id
       end
 
+      body = builder_to_xml(builder)
```

Comment:
not related to this task but found a cleaner way to work with nokogiri. could be useful in other bgs calls.